hawkular-metrics/hawkular-metrics-liveness.py

```diff
@@ -26,14 +26,16 @@
 timeout = os.environ.get("STARTUP_TIMEOUT", 500)
 
 uptime = os.popen("ps -eo comm,etimes | grep -i standalone.sh | awk '{print $2}'").read()
+if type(uptime) == str:
+  uptime = 0
```

Comment:
The value returned from os.popen(...).read() is always a string. This means we are hardcoding the uptime value to be 0 all the time.

What should be done here is to check if the value returned is an empty string (eg "") and if so, to set it to 0